tastypie/resources.py

```diff
@@ -270,6 +272,18 @@ def wrapper(request, *args, **kwargs):
 
         return wrapper
 
+    def handle_patch_for_filefields(self, bundle, data_to_patch):
+        obj = bundle.obj
+        data_to_patch_copy = data_to_patch.copy()
+        fixed_data = {}
+
+        for field in obj._meta.local_fields:
```

Comment:
This code presumes the Django ORM is being used, which may not be the case for the Resource class.